cms/djangoapps/contentstore/utils.py

```diff
@@ -147,6 +147,26 @@ def get_lms_link_for_about_page(course_key):
     )
 
 
+# pylint: disable=invalid-name
+def get_lms_link_for_certificate_web_view(user_id, course_key, mode):
+    """
+    Returns the url to the certificate web view.
+    """
+    assert isinstance(course_key, CourseKey)
+
+    if settings.LMS_BASE is not None:
+        certificate_web_base = settings.LMS_BASE
+    else:
+        return None
+
+    return u"//{certificate_web_base}/certificates/user/{user_id}/course/{course_id}?preview={mode}".format(
+        certificate_web_base=certificate_web_base,
+        user_id=user_id,
+        course_id=course_key.to_deprecated_string(),
```

Comment:
I believe to_deprecated_string() is no longer supported -- you should instead use unicode(course_key)